### PR TITLE
fix(InputWrapper) Using 'node' prop-type for label prop in InputWrapper.

### PR DESCRIPTION
**Pull request by SeaOfTea**

### CODE REVIEW

**Comment by lpoulter** [comment]
Can you add some description, what the idea here? What other types of label would you be setting?

**Comment by SeaOfTea** [comment]
Hey @lpoulter , yeah sorry. I did have some description in here, but realised it was a link to invision, and as axiom is a public repo I wasn't 100% sure it was a good idea to have it in the description. Was in a rush so just removed the description altogether. The reasoning for this PR is that a design requires a clickable link that will change some state (in my case hiding a textarea).

screen shot 2018-07-27 at 16 06 31

I hadn't thought about possibly using oneOf, maybe `oneOf([PropTypes.object, PropTypes.string])` ?

**Comment by pmsorhaindo** [comment]
I think `oneOf([PropTypes.string, PropTypes.node])` is what's needed here.

File: packages/axiom-components/src/Form/InputWrapper.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ export default class InputWrapper extends Component {
   static propTypes = {
     children: PropTypes.node,
     isTarget: PropTypes.bool,
-    label: PropTypes.string,
+    label: PropTypes.any,
```
Comment (HHogg) [inline packages/axiom-components/src/Form/InputWrapper.js]: `PropTypes.node` as it would cover strings and React elements.

**Comment by SeaOfTea** [comment]
Have updated this PR to use `node` instead of `any`.

**Pull request merged**